includes/Stories_Renderer/Renderer.php

```diff
@@ -340,6 +341,7 @@ protected function get_block_classes() {
 
 		$block_classes   = [];
 		$block_classes[] = 'web-stories-list';
+		$block_classes[] = ( $this->is_amp_request() ) ? 'is-amp' : '';
```

Comment:
Instead of adding an `is-amp` class, the `html` element will have an `amp` attribute already, which you can target via CSS.